pkg/status/status.go

```diff
@@ -127,50 +165,23 @@ func (s *status) setOperandVersion() {
 	// Report the operator version
 	operatorVersion := configv1.OperandVersion{
 		Name:    "operator",
-		Version: version.Version,
+		Version: os.Getenv("RELEASE_VERSION"),
```

Comment:
I'm hesitant about using `os.Getenv` inside this method. It will make testing more complicated, among other things. Can we do the `os.Getenv` in the `main()` method and compose the status struct to include it as a variable?

Reply:
Updated accordingly.